### DISCUSSION TITLE
Domotizzare casa da zero

### DISCUSSION THREAD

**Discussion by Lucacali87** [indomus/forum#3160] [Nuove idee, proposte, progetti e guide]
Ciao a tutti, sono nuovo nel mondo domotica e vorrei qualche consiglio su come iniziare.
Sto acquistando casa e quindi prima di decidere le prese elettriche vorrei anche progettare un minimo di domotica. La casa è già dotata di climatizzatori, tapparelle elettriche, l'illuminazione invece dovrò farla io. Inoltre c'è impianto fotovoltaico con accumulo e pavimento riscaldato ma al momento non ho tutti i dettagli.
La mia idea è quella di gestire tapparelle, climatizzatori, luci(accensione con sensore movimento, scenari etc) da un monitor all'ingresso e da app del telefono. Poi magari integro anche fotovoltaico, serratura, pavimento etc... diciamo sto facendo uno studio di fattibilità.
L'idea sarebbe usare home assistant su raspberry o simile ed usare un tablet a parete. Pensate possa essere la scelta migliore? Alternative?
Non avendo esperienza in merito faccio fatico a progettarla da zero

**Comment by indomus** [comment]
Ottima idea quella di partire prima dall'HUB, bravo, si fa precisamente così.
https://indomus.it/formazione/primi-passi-con-home-assistant-da-dove-cominciare

**Reply by Lucacali87** [reply]
Ho letto praticamente tutte le guide :D Quindi potrebbe essere la soluzione giusta? 
Ma poi servirà un hub per le tapparelle, uno per le luci etc... o sono in grado di comunicare direttamente con home assistant?

**Reply by indomus** [reply]
tendenzialmente fai tutto da hub, specie se usi zigbee, z-wave, thread, matter.

**Reply by Lucacali87** [reply]
ma le luci per esempio, se volessi averle drimmerabili ed automatiche tramite sensore, bisogna prendere la luce smart, la presa smart o usare uno shelby? In modo da poterle ovviamente modificare anche da pulsante (quindi se chiudo da pulsante poi dopo devono riaccendersi). Sto cercando di smarcare un pò di dubbi

**Reply by indomus** [reply]
dipende. ogni scenario cambia, ogni punto luce può avere delle caratteristiche diverse. tendenzialmente si utilizzano attuatori come gli shelly o i sonoff

**Reply by Lucacali87** [reply]
quindi lampada led classica e shelly può andare bene per quello che voglio fare?
